docs/PowerAuth-Server-1.5.0.md

```diff
@@ -127,3 +127,52 @@ Since version `1.5.0`, MySQL database is not supported anymore.
 
 PostgreSQL JDBC driver is already included in the WAR file.
 Oracle JDBC driver remains optional and must be added to your deployment if desired.
+
+## Configuration properties
```

Comment:
Wrong resolution of merge conflict, fixed by https://github.com/wultra/powerauth-server/pull/1000